src/render/vertex_array_object.js

```diff
@@ -14,9 +14,7 @@ class VertexArrayObject {
     boundPaintVertexBuffers: Array<VertexBuffer>;
     boundIndexBuffer: ?IndexBuffer;
     boundVertexOffset: ?number;
-    boundDynamicVertexBuffer: ?VertexBuffer;
-    boundDynamicVertexBuffer2: ?VertexBuffer;
-    boundDynamicVertexBuffer3: ?VertexBuffer;
+    boundDynamicVertexBuffers: Array<?VertexBuffer>;
```

Comment:
Same as below, do we need to support `null`/ `undefined`?
```suggestion
    boundDynamicVertexBuffers: Array<VertexBuffer>;
```